src/erlcloud_lambda.erl

```diff
@@ -761,6 +761,8 @@ decode_body(<<>>) ->
 decode_body(BinData) ->
     jsx:decode(BinData).
```

Comment:
seem to me having decode removed under an option is needed to, no?

Reply:
Fixed.  `raw_response_body` option is added